src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountExtractPdpEntryServiceImpl.java

```diff
@@ -0,0 +1,139 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import java.sql.Date;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.kfs.kns.service.DataDictionaryService;
+import org.kuali.kfs.pdp.PdpConstants;
+import org.kuali.kfs.pdp.businessobject.PaymentGroup;
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+import edu.cornell.kfs.concur.ConcurConstants;
+import edu.cornell.kfs.concur.batch.businessobject.ConcurStandardAccountingExtractDetailLine;
+import edu.cornell.kfs.concur.batch.service.ConcurStandardAccountExtractPdpEntryService;
+import edu.cornell.kfs.concur.batch.xmlObjects.PdpFeedAccountingEntry;
+import edu.cornell.kfs.concur.batch.xmlObjects.PdpFeedDetailEntry;
+import edu.cornell.kfs.concur.batch.xmlObjects.PdpFeedGroupEntry;
+import edu.cornell.kfs.concur.batch.xmlObjects.PdpFeedHeaderEntry;
+import edu.cornell.kfs.concur.batch.xmlObjects.PdpFeedPayeeIdEntry;
+
+public class ConcurStandardAccountExtractPdpEntryServiceImpl implements ConcurStandardAccountExtractPdpEntryService {
+    
+    protected DataDictionaryService dataDictionaryService;
+    
+    private Integer payeeNameFieldSize;
+
+    @Override
+    public PdpFeedHeaderEntry buildPdpFeedHeaderEntry(Date batchDate) {
+        PdpFeedHeaderEntry headerEntry = new PdpFeedHeaderEntry();
+        headerEntry.setChart(ConcurConstants.StandardAccountingExtractPdpConstants.DEFAULT_CHART_CODE);
+        headerEntry.setCreationDate(formatDate(batchDate));
+        headerEntry.setSubUnit(ConcurConstants.StandardAccountingExtractPdpConstants.DEFAULT_SUB_UNIT);
+        headerEntry.setUnit(ConcurConstants.StandardAccountingExtractPdpConstants.DEFAULT_UNIT);
+        return headerEntry;
+    }
+
+    @Override
+    public PdpFeedGroupEntry buildPdpFeedGroupEntry(ConcurStandardAccountingExtractDetailLine line) {
+        PdpFeedGroupEntry currentGroup = new PdpFeedGroupEntry();
+        currentGroup.setPayeeName(buildPayeeName(line.getEmployeeLastName(), line.getEmployeeFirstName(), 
+                line.getEmployeeMiddleInitital()));
+        currentGroup.setPayeeId(buildPayeeIdEntry(line));
+        currentGroup.setPaymentDate(formatDate(line.getBatchDate()));
+        currentGroup.setCombineGroupInd(ConcurConstants.StandardAccountingExtractPdpConstants.COMBINDED_GROUP_INDICATOR);
+        currentGroup.setBankCode(ConcurConstants.StandardAccountingExtractPdpConstants.BANK_CODE);
```

Comment:
I don't know for certain, I was just plugging in values Sherry defined in the PDP out pul file requirements   I can track it down further